app/controllers/api/v1/timeframe_versions_controller.rb

```diff
@@ -0,0 +1,45 @@
+class Api::V1::TimeframeVersionsController < Api::BaseController
+  skip_before_action :verify_authenticity_token
+  before_action :set_page, only: :index
+
+  def index
+    if from_time.nil? || to_time.nil?
+      render plain: 'timeframe parameters must be iso8601 formatted',
+             status: :bad_request
+    else
+      render_rubygems(version_timeframe_query)
+    end
+  end
+
+  private
+
+  def from_time
+    @parse_from_time ||= Time.iso8601(params.require(:from))
+  rescue ArgumentError
+    return
+  end
+
+  def to_time
+    @to_time ||= params[:to].blank? ? Time.zone.now : Time.iso8601(params[:to])
+  rescue ArgumentError
+    return
+  end
+
+  def version_timeframe_query
```

Comment:
Can we please move this to `Version` model and add unit tests for it? Also, perhaps a better name could be `created_between`.

Reply:
Sure thing, will do 👍